Makefile

```diff
@@ -0,0 +1,61 @@
+DB_SCRIPTS_IMAGE_NAME = 10.40.71.55:5000/rsww_184529_db_scripts
+
+# ran on a dev machine
+.PHONY: dev-up
+dev-up: build-dev
+	# doing a weird set -m / fg dance to allow to
+	# still stop the whole stack easily during dev by Ctrl+C
+	set -m ; \
+	$(MAKE) dev-up-only & \
+	$(MAKE) dev-fill-db \
+		|| { exit_code=$$?; kill \%1; fg; exit "$$exit_code"; } ; \
+	fg
+
+# ran on a dev machine
+.PHONY: dev-up-only
+dev-up-only:
+	docker compose -f compose.yaml -f compose.dev.yaml up
+
+# ran on a dev machine
+.PHONY: dev-up-only-daemon
+dev-up-only-daemon: build-dev
```

Comment:
There is slight name inconsistency between this target and dev-up-only, since one rebuilds all the container and the other is not